labstats/bin/init.py

```diff
@@ -55,12 +55,15 @@
     SELECT `user`, `host`, `start` FROM `session` WHERE `end` IS NULL;
 
 CREATE VIEW `users_in_lab_count_public` AS
-    SELECT count(*) as `count` FROM `users_in_lab`;
+    SELECT COUNT(DISTINCT `user`) AS `count` FROM `users_in_lab`;
 
 CREATE VIEW `staff_in_lab_public` AS
     SELECT `user`, `host`, `start` FROM `users_in_lab` WHERE `user` IN (
         SELECT `user` FROM `staff`
     );
+    
+CREATE VIEW `staff_in_lab_count_public` AS
+    SELECT COUNT(DISTINCT `user`) AS `count` FROM `staff_in_lab_public`; 
```

Comment:
I'm conflicted on this too, since just selecting users from `staff_in_lab_public` seems easy enough, but I'm leaning more towards just having this logic in ocflib instead since it's such a simple query and there aren't going to be any performance issues since there will only be a certain number of users in the lab at a time.